independent-publisher-connectors/WorldTime/apiDefinition.swagger.json

```diff
@@ -0,0 +1,318 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "A simple API to get the current time based on a request with a timezone.",
+    "title": "WorldTime",
+    "version": "1.0",
+    "contact": {
+      "name": "Fördős András",
+      "email": "fordosa90@gmail.com"
+    }
+  },
+  "host": "worldtimeapi.org",
+  "basePath": "/api/",
+  "schemes": [
+    "http"
+  ],
+  "paths": {
+    "/ip/{ipv4}": {
+      "get": {
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "ipv4",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "IPv4 Address",
+            "x-ms-url-encoding": "single",
+            "description": "A valid IPv4 address, such as 8.8.8.8"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/DateTimeJsonResponse"
+            }
+          },
+          "default": {

```

Comment:
We do not accept default responses. Please update all responses accordingly.

Reply:
@v-eisljami @Amjed-Ayoub pushed an update, thank you for the notification